src/cli/discord.ts

```diff
@@ -53,9 +53,25 @@ function filterEnv (env: NodeJS.ProcessEnv): NodeJS.ProcessEnv {
   return res
 }
 
-async function findDiscord (): Promise<string> {
-  // todo
-  return '/opt/discord-canary/DiscordCanary'
+async function findDiscord (): Promise<string | null> {
+  let path = null
+
+  if (process.platform === 'win32') {
+    // get discord canary path
+    const discordPath = join(process.env.LOCALAPPDATA!, 'DiscordCanary')
+
+    const discordDirectory = await readdir(discordPath)
+    // get the current build folder
+    const currentBuild = discordDirectory
+      .filter((appPath) => appPath.startsWith('app-'))
+      .reverse()[0]
+
+    // append discord canary Executable path
+    path = join(discordPath, currentBuild, 'DiscordCanary')
+  } else if (process.platform === 'linux') {
+    path = '/usr/share/discord-canary/DiscordCanary'
```

Comment:
This still needs to be reverted to what it was before.

Reply:
you mean arch's main folder?

Reply:
nvm I changed it to '/opt/discord-canary/DiscordCanary'